<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changestu</class>
 <widget class="QDialog" name="changestu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/纯色.gif);</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>修改学生信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>语文成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stus_Cmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>地理成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stum_Gmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>英语成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stum_Emark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>数学成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stus_Mmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>英语成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stus_Emark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>数学成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stum_Mmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>英语成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stul_Emark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>语文成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stum_Cmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>100</width>
     <height>98</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/生气鲨鱼.gif);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,8">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>专业：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="major">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>计算机类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机械类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>自动化类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电子信息类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>经济学类</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>71</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>学号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lid">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="maxLength">
         <number>9</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>请输入学号（9位）</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8,0">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lname">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="placeholderText">
         <string>请输入姓名</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="male">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>男</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="female">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>女</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,6,12">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>年龄：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lage">
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,6,12">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>班级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lstuclass">
        <item>
         <property name="text">
          <string>2101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2001</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2002</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1901</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>*根据在读状况填写成绩：（未填写则默认为0分！）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="stum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>中学生</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="5,2,5">
    <item>
     <widget class="QPushButton" name="btnok">
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btncancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>历史成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stum_Hmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="stul">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>大学生</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>程序成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stul_Pmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="stus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>小学生</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,1">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>高数成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stul_Mmark">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_8</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget_11</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_9</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>stum</zorder>
  <zorder>layoutWidget_10</zorder>
  <zorder>layoutWidget_12</zorder>
  <zorder>stul</zorder>
  <zorder>layoutWidget_13</zorder>
  <zorder>stus</zorder>
  <zorder>layoutWidget_14</zorder>
  <zorder>widget</zorder>
 </widget>
 <tabstops>
  <tabstop>lid</tabstop>
  <tabstop>lname</tabstop>
  <tabstop>male</tabstop>
  <tabstop>female</tabstop>
  <tabstop>lage</tabstop>
  <tabstop>lstuclass</tabstop>
  <tabstop>stus</tabstop>
  <tabstop>stus_Emark</tabstop>
  <tabstop>stus_Mmark</tabstop>
  <tabstop>stus_Cmark</tabstop>
  <tabstop>stum</tabstop>
  <tabstop>stum_Emark</tabstop>
  <tabstop>stum_Mmark</tabstop>
  <tabstop>stum_Cmark</tabstop>
  <tabstop>stum_Gmark</tabstop>
  <tabstop>stum_Hmark</tabstop>
  <tabstop>stul</tabstop>
  <tabstop>major</tabstop>
  <tabstop>stul_Emark</tabstop>
  <tabstop>stul_Pmark</tabstop>
  <tabstop>stul_Mmark</tabstop>
  <tabstop>btnok</tabstop>
  <tabstop>btncancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
